src/mainActions.ts

```diff
@@ -1,144 +1,169 @@
 }
 
-export function importJsonFile(mainWindow: BrowserWindow){
+export function importJsonFile(mainWindow: BrowserWindow) {
   console.log("Prompt to Import JSON file");
-  dialog.showOpenDialog(mainWindow, {    
-    title: 'Open a JSON File',
-    filters: [{
-        name: 'JSON File',
-        extensions: ['json']
-    }]
-    }).then(({filePaths}) => {
+  dialog
+    .showOpenDialog(mainWindow, {
+      title: "Open a JSON File",
+      filters: [
+        {
+          name: "JSON File",
+          extensions: ["json"],
+        },
+      ],
+    })
+    .then(({ filePaths }) => {
       if (filePaths === undefined || !filePaths[0]) {
         return;
       }
       chooseSqlitePathForJsonImport(mainWindow, filePaths[0]);
     });
 }
 
-export function openSqliteFile(mainWindow: BrowserWindow){
+export function openSqliteFile(mainWindow: BrowserWindow) {
   console.log("Prompt to open SQLite file");
 
-  dialog.showOpenDialog(mainWindow, {
-    properties: ['openFile'],
-    title: 'Open a SQLite File',
-    filters: [{
-      name: 'SQLite File',
-      extensions: ['sqlite']
-    }]
-  }).then(({ filePaths }) => {
-    if (filePaths && filePaths[0]) {
-      const sqliteFilePath = filePaths[0]
-      const reply: SQLITE_IMPORT_REPLY_FORMAT = { sqliteFilePath };
-      mainWindow.webContents.send(IMPORT_REPLY_CHANNEL.SQLITE, reply);
-    } else {
-      console.log("Sqlite file path isn't valid:", filePaths);
-      return;
-    }
-  });
+  dialog
+    .showOpenDialog(mainWindow, {
+      properties: ["openFile"],
+      title: "Open a SQLite File",
+      filters: [
+        {
+          name: "SQLite File",
+          extensions: ["sqlite"],
+        },
+      ],
+    })
+    .then(({ filePaths }) => {
+      if (filePaths && filePaths[0]) {
+        const sqliteFilePath = filePaths[0];
+        const reply: SQLITE_IMPORT_REPLY_FORMAT = { sqliteFilePath };
+        mainWindow.webContents.send(IMPORT_REPLY_CHANNEL.SQLITE, reply);
+      } else {
+        console.log("Sqlite file path isn't valid:", filePaths);
+        return;
+      }
+    });
 }
 
-export function saveSqliteFile(mainWindow: BrowserWindow){
+export function saveSqliteFile(mainWindow: BrowserWindow) {
   console.log("Save sqlite file prompt");
 
-  dialog.showSaveDialog(mainWindow, {
-    title: 'Save as a Database File',
-    defaultPath: 'fileName.sqlite',
-    filters: [
-      { name: 'SQLite File', extensions: ['sqlite'] }
-    ]
-  }).then((file) => {
-    const sqliteFilePath = file?.filePath;
-    if (sqliteFilePath) {
-      const reply: SQLITE_SAVE_REPLY_FORMAT = { sqliteFilePath };
-      mainWindow.webContents.send(SAVE_REPLY_CHANNEL.SQLITE, reply);
-    } else {
-      console.log("Sqlite file path isn't valid:", file, sqliteFilePath);
-      return;
-    }
-  });
+  dialog
+    .showSaveDialog(mainWindow, {
+      title: "Save as a Database File",
+      defaultPath: "fileName.sqlite",
+      filters: [{ name: "SQLite File", extensions: ["sqlite"] }],
```

Comment:
use multiple lines like above for more readable code whenever possible

Reply:
yep, usually i let eslint do the magic :p

Reply:
Yup, that's the best way, to leave these to linters, saves time.